src/algebra/category/Group/adjunctions.lean

```diff
@@ -73,6 +77,21 @@ example {G H : AddCommGroup.{u}} (f : G ⟶ H) [mono f] : function.injective f :
 
 end AddCommGroup
 
+namespace Group
+
+def free : Type u ⥤ Group :=
```

Comment:
The linter is going to want docstrings for these defs.

Reply:
Thanks a lot for the thorough review. Provided.